Enforce make for MSYS. Needed for config.tests outside qtbase,
e.g. libtiff in qtimageformats.
Cf. https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-qt5/0025-qt-5.8.0-force-using-make-on-msys.patch
--- a/mkspecs/features/configure_base.prf
+++ b/mkspecs/features/configure_base.prf
@@ -10,7 +10,7 @@
 !isEmpty(QMAKE_MAKE) {
     # We were called recursively. Use the same make.
 } else: if(equals(MAKEFILE_GENERATOR, UNIX)|equals(MAKEFILE_GENERATOR, MINGW)) {
-    !equals(QMAKE_HOST.os, Windows): \
+    !isEmpty(QMAKE_SH)|!equals(QMAKE_HOST.os, Windows): \
         QMAKE_MAKE = make
     else: \
         QMAKE_MAKE = mingw32-make
